bears/general/AnnotationBear.py

```diff
@@ -28,104 +27,203 @@ def run(self, filename, file, language: str, coalang_dir: str = None):
                                 respectively.
         """
         lang_dict = LanguageDefinition(language, coalang_dir=coalang_dir)
-        # Strings
-        # TODO treat single-line and multiline strings differently
-        strings = dict(lang_dict["string_delimiters"])
-        strings.update(lang_dict["multiline_string_delimiters"])
-        strings_found = self.find_with_start_end(filename, file, strings)
-        # multiline Comments
-        comments_found = self.find_with_start_end(
-                filename, file, dict(lang_dict["multiline_comment_delimiters"]))
-        # single-line Comments
-        comments_found.update(self.find_singleline_comments(
-                          filename, file, list(lang_dict["comment_delimiter"])))
-
-        matches_found = strings_found | comments_found
-        # Remove Nested
-        unnested_annotations = set(filter(
-                                       lambda arg: not starts_within_ranges(
-                                           arg, matches_found),
-                                       matches_found))
-        # Yield different results for strings and comments
-        strings_found = tuple(filter(lambda arg: arg in unnested_annotations,
-                                     strings_found))
-        comments_found = tuple(filter(lambda arg: arg in unnested_annotations,
-                                      comments_found))
-        yield HiddenResult(self, {'comments': comments_found,
-                                  'strings': strings_found})
+        single_strings = dict(lang_dict["string_delimiters"])
```

Comment:
please put "delimiters" in the variable name. Variable names should be self-explaining in most cases.
"single_strings" sounds like a list or tuple of strings that don't have a girlfriend. The only way I know what's in there is that `lang_dict["string_delimiters"]` is explaining that really well. Why not just call this variable `string_delimiters`?
goes for the following lines as well ofc.